lm-core/src/main/contraband/librarymanagement.json

```diff
@@ -733,6 +733,19 @@
         { "name": "devConnection", "type": "String?",     "default": "None", "since": "0.0.1" }
       ]
     },
+    {
+      "name": "LicenseInfo",
```

Comment:
This probably should be called License. Note that we already have License object that lists popular licenses. We should let the companion object expand License Extra so we don't lose that.